test/common/config/grpc_mux_impl_test.cc

```diff
@@ -76,8 +70,8 @@ class GrpcMuxImplTest : public testing::Test {
   NiceMock<Event::MockDispatcher> dispatcher_;
   Runtime::MockRandomGenerator random_;
   Grpc::MockAsyncClient* async_client_;
-  Event::MockTimer* timer_;
-  Event::TimerCb timer_cb_;
+  Event::MockTimer* timer_; // Only used by ResetStream
```

Comment:
This looks like a nice cleanup. Can you just move these two variables to be local to `ResetStream`? This will make reasoning even simpler.

Reply:
done.